### ISSUE TITLE
Deploy branch

### ISSUE THREAD

**Issue by willingc** [nteract/nteract.github.io#29]
@lgeiger @rgbkrk @captainsafia Howdy! What is the preferred approach for deploying to production? 

It looks like we are deploying from Travis. Though, I'm not seeing the latest changes. 

I'm assuming this is no longer relevant since master is pretty different than develop at this point. 
```
After building and when you're happy with things run git subtree push --prefix dist origin master
```

**Comment by lgeiger** [comment]
I bet there is a token missing on Travis. I'll take a look.

**Comment by captainsafia** [comment]
@lgeiger The last time I checked this was due to an issue with building node-sass. I couldn't figure it out at the time so I let it be.

What I do at the moment @willingc is run the `deploy.js` file locally on my machine, making sure that the configuration looks like this.

```
ghpages.publish(path.join(__dirname, 'dist'), {
  repo: 'https://github.com/nteract/nteract.github.io.git',
  branch: 'master',
});
```

**Comment by willingc** [comment]
Let's see if we can get the deploy working as it would save time.

@captainsafia Which branch is the production branch: `master` or `develop`?

**Comment by lgeiger** [comment]
@captainsafia The build issue was due to a outdated node version on Travis and is resolved since #24.

#30 should make deploying via Travis work again. Travis needed a token to be able to push to this repo.

**Comment by captainsafia** [comment]
> @captainsafia Which branch is the production branch: master or develop?

`master` is the one we want to deploy to.

> @captainsafia The build issue was due to a outdated node version on Travis and is resolved since #24.

> #30 should make deploying via Travis work again. Travis needed a token to be able to push to this repo.

Ah! I see. Thanks! I'll merge those PRs.

**Comment by willingc** [comment]
Thanks @captainsafia @lgeiger 👍

**Comment by lgeiger** [comment]
Hmm the deploy by Travis didn't work. I'll try to do it manually and see if it works.

**Comment by lgeiger** [comment]
I deployed manually. Now let's figure out what's the problem with Travis.

**Comment by captainsafia** [comment]
> I deployed manually. Now let's figure out what's the problem with Travis.

Travis is a naughty boy and needs a loooong time out!

**Comment by willingc** [comment]
Oooh... working now :tada:

Nicely done @lgeiger and @captainsafia

**Comment by lgeiger** [comment]
@willingc You are seeing the manual deploy.

Travis still doesn't work. I'm investigating. 😕

**Comment by lgeiger** [comment]
Alright, it's fixed by #32.

We now have automatic deploys :tada:
